trace/opencensus/trace/config_integration.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+import logging
+
+log = logging.getLogger(__name__)
+
+DEFAULT_INTEGRATIONS = {
+    'mysql': True,
```

Comment:
Framework integration should be separated from this. Because the usage is different, user needs to use the middleware to config the framework integrations.